collections/landsat-8-l1.yaml

```diff
@@ -150,5 +150,232 @@ CubeDimensions:
       - SZA
       - dataMask
 sci:citation: "Landsat 8 product courtesy of the U.S. Geological Survey, processed with Sentinel Hub"
+Summaries:
+  instruments:
+    - oli
+    - tirs
+  sat:orbit_state:
+    - ascending
+    - descending
+  eo:bands:
+    - center_wavelength: 0.44296
+      common_name: coastal
+      description: Ultra Blue (443 nm)
```

Comment:
Would everything work with the `description` fields? Other collections don't have this field for `eo:bands` or any other bands (`sar:bands` or `bands`)